### PR TITLE
OSDOCS-5306: changing authToken to FORWARD

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-5306
Link to docs preview:

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
@stleerh @memodi @jotak This PR is ready for your review.

**Review by jotak — APPROVED** [review]
lgtm, thanks!

**Comment by memodi** [comment]
@skrthomas - moving to FORWARD mode, we also need to update: https://docs.openshift.com/container-platform/4.12/networking/network_observability/installing-operators.html#network-observability-roles-create_network_observability to remove ClusterRoleBinding for netobserv-plugin SA,  @jotak - could you confirm?

```
- kind: ServiceAccount
  name: netobserv-plugin               
  namespace: netobserv
```

**Review by stleerh — APPROVED** [review]
/lgtm

**Comment by stleerh** [comment]
> @skrthomas - moving to FORWARD mode, we also need to update: https://docs.openshift.com/container-platform/4.12/networking/network_observability/installing-operators.html#network-observability-roles-create_network_observability to remove ClusterRoleBinding for netobserv-plugin SA, @jotak - could you confirm?
> 
> ```
> - kind: ServiceAccount
>   name: netobserv-plugin               
>   namespace: netobserv
> ```

Mehul is correct. With FORWARD, these three lines are not needed.

**Comment by stleerh** [comment]
> @skrthomas - moving to FORWARD mode, we also need to update: https://docs.openshift.com/container-platform/4.12/networking/network_observability/installing-operators.html#network-observability-roles-create_network_observability to remove ClusterRoleBinding for netobserv-plugin SA, @jotak - could you confirm?
> 
> ```
> - kind: ServiceAccount
>   name: netobserv-plugin               
>   namespace: netobserv
> ```

Mehul is correct. With FORWARD, these three lines are not needed.

File: modules/network-observability-roles-create.adoc [hunk 1]
```diff
@@ -41,9 +41,5 @@ subjects:
 - kind: ServiceAccount
   name: flowlogs-pipeline              <1>
   namespace: netobserv
-- kind: ServiceAccount
```
Comment (skrthomas) [inline modules/network-observability-roles-create.adoc:44]: @memodi removed this info

**Pull request merged**